packages/turf-collect/index.js

```diff
@@ -27,20 +29,38 @@ var inside = require('@turf/inside');
  *
  * collected.features[0].properties.values // => [200, 600]);
  */
-module.exports = function collect(polygons, points, inProperty, outProperty) {
-    polygons.features.forEach(function (poly) {
-        var values = points.features.filter(function (pt) {
-            return inside(pt, poly);
-        }).map(function (pt) {
-            return pt.properties[inProperty];
-        });
+
+module.exports = function (polyFC, pointFC, propertyIn, propertyOut) {
```

Comment:
Why change the input variable names? I personally prefer `polygons` over `polyFC`.

If you are changing input variable names, then you also have to change the JSDocs (`@param`).